src/sbd-inquisitor.c

```diff
@@ -1307,23 +1303,9 @@ int main(int argc, char **argv, char **envp)
         } else if (strcmp(argv[optind], "ping") == 0) {
             exit_status = ping_via_slots(argv[optind + 1], servants_leader);
 
-        } else if (strcmp(argv[optind], "watch") == 0) {
-                if(disk_count > 0) {
-                    /* If no devices are specified, its not an error to be unable to find one */
-                    open_any_device(servants_leader);
```

Comment:
timeout_watchdog_warn calculation below doesn't work anymore without open_any_device before I'm afraid.
question is do we need it anywhere else than for the watch-command?
If yes why not move there below.
Then we can go for a single if () { } else if () { } structure including query-watchdog and test-watchdog and just set 'exit_status = -2' if it is none of the known commands.

Reply:
Hm I've missed that and even the comment says it's read from open_any_device.
From looking at the code it looks like it's only used for the watch-command in inquisitor_child function. 
But will investigate a bit more.

Reply:
Well, was broken a bit before already. This is a good occasion to clean up instead of breaking it further ;-)